src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+  const totalCost = days * 40;
+
+  switch (true) {
+    case (days >= 7):
+      cost = totalCost - 50;
+      break;
+    case (days >= 3):
+      cost = totalCost - 20;
+      break;
+    default:
+      cost = totalCost;
```

Comment:
Switching true is not the best idea. 
You can just check if days are enough for some discount and immediately return the full price.
Would be great to create variables for 3, 7, 20 and 50 to make your code more descriptive.
Also, do you really need `cost` variable?